omod/src/main/java/org/openmrs/web/controller/patient/MergePatientsFormController.java

```diff
@@ -101,7 +103,11 @@ protected ModelAndView onSubmit(HttpServletRequest request, HttpServletResponse
 			}
 			catch (APIException e) {
 				log.error("Unable to merge patients", e);
-				httpSession.setAttribute(WebConstants.OPENMRS_ERROR_ATTR, "Patient.merge.fail");
+                String message = e.getMessage();
+                if(message == null || "".equals(message)){
```

Comment:
Creating a new PR with the updated ticket info.
I will make these changes also.

New PR https://github.com/openmrs/openmrs-module-legacyui/pull/75.